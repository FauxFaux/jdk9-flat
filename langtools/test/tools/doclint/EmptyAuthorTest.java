/*
 * @test /nodynamiccopyright/
 * @build DocLintTester
 * @run main DocLintTester -Xmsgs:-syntax EmptyAuthorTest.java
 * @run main DocLintTester -Xmsgs:syntax -ref EmptyAuthorTest.out EmptyAuthorTest.java
 */

/** @author */
public class EmptyAuthorTest {
}
