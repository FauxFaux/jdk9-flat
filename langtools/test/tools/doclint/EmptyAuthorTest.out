EmptyAuthorTest.java:8: warning: no description for @author
/** @author */
    ^
1 warning

